5g-control-plane/templates/configmap-upf-adapter.yaml

```diff
@@ -13,6 +13,12 @@
 {{- if not (hasKey $upfadaptercfg "mongodb") -}}
 {{- $_ := dict "name" .Values.config.mongodb.name "url" .Values.config.mongodb.url | set $upfadaptercfg "mongodb" -}}
 {{- end }}
+{{- if not (hasKey $upfadaptercfg "logger") -}}
+{{- $_ := .Values.config.logger | set $upfadaptercfg "logger" -}}
+{{- end }}
+{{- if not (hasKey $upfadaptercfg.configuration.kafkaInfo "enableKafka") -}}
```

Comment:
I do not see this parameter in the `upfadapter`. Does it mean that you are going to add it as part of the enabling metrics?

Reply:
I shared Helm chart changes first...I am testing upf adapter changes and once confident then I can raise that PR. Helm chart merge will not break anything.